src/go/k8s/Makefile

```diff
@@ -76,11 +78,25 @@ docker-push:
 push-to-kind:
 	kind load docker-image ${IMG}
 
+# Execute end to end tests
+e2e-tests: kuttl docker-build
+	$(KUTTL) test
+
 # Download controller-gen locally if necessary
 CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
 controller-gen:
 	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1)
 
+# Download kuttl locally if necessary
+KUTTL = $(shell pwd)/bin/kubectl-kuttl
+kuttl:
+	$(call go-get-tool,$(KUTTL),github.com/kudobuilder/kuttl/cmd/kubectl-kuttl@v0.8.0)
```

Comment:
why is the way kuttl is installed different from crlfmt? 🤔

Reply:
Turns out that `rpk` anchor different version of `crlfmt` than what is downloaded via `go get`. Because of that [lint all codez pipeline](https://github.com/vectorizedio/redpanda/blob/dev/.github/workflows/lint.yml#L45) failed when I formatted it using newer version from `go get`. I tried to bump the version in `rpk` https://github.com/vectorizedio/redpanda/pull/552 which failed miserably because of the `tabwriter` dependency. The tests are validating exact output which has white characters.

BTW latest version of `crlfmt` seems to better align with the `go fmt` Reference https://github.com/cockroachdb/crlfmt/commit/cae1965cbdf28291a28715f9b91a7a4061c1c8ed

And the whole diff ours vs latest version
https://github.com/cockroachdb/crlfmt/compare/b9cb16fe9a33c5c41e8d5ff50a7772368d3398dd...b3eff0b87c79b24fa4f5502dfbf8ec68561594c9